roles/etcd/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Install etcd
+  yum: pkg=etcd state=installed disable_gpg_check=yes
+
+- name: Write etcd global config file
+  template: src=etcd.conf.j2 dest=/etc/etcd/etcd.conf
```

Comment:
To be clear, I thought we were mainly trying to prevent things like:
```
template:
  src: {{ item }}
  dest: /etc/{{ item }}
with_items: [ item1, item2, item3 ]
```